packages/fx-core/src/plugins/resource/bot/v2/mapping.ts

```diff
@@ -61,23 +58,23 @@ export function getRuntime(lang: ProgrammingLanguage): Runtime {
   if (runtime) {
     return runtime;
   }
-  return defaultRuntime;
+  throw new Error("invalid bot input");
```

Comment:
Maybe we could make this string "invalid bot input" as a const?

Reply:
Done.